app/views/comments/_form.html.erb

```diff
@@ -1,17 +1,14 @@
-<%= form_with model: [ @article, @article.comments.build ] do |form| %>
+<%= simple_form_for [@article, @article.comments.build] do |form| %>
   <div>
     <%= form.label :commenter %><br>
-    <%= form.text_field :commenter %>
+    <%= form.input_field :commenter %>
```

Comment:
If you're using simple form, use `form.input :commenter, as: ...` if you need to specify the type of field.
If it's just input, no need to specify it.